pkg/antctl/raw/multicluster/deploy/deploy_helper.go

```diff
@@ -121,7 +121,21 @@ func createResources(cmd *cobra.Command, apiGroupResources []*restmapper.APIGrou
 			if !kerrors.IsAlreadyExists(err) {
 				return err
 			}
-			fmt.Fprintf(cmd.OutOrStdout(), "%s/%s already exists\n", unstructuredObj.GetKind(), unstructuredObj.GetName())
+			existingRes, err := dri.Get(context.TODO(), unstructuredObj.GetName(), metav1.GetOptions{})
```

Comment:
Should we just return if err == nil?

Reply:
Do you mean return here immediately if there is an existing CRD resource?
If we return here, it won't do anything even the manifest is changed. I think we need to call `dri.Update` to apply the change.

Reply:
I wrote return when `err == nil`, not when the CR exists (which should have returned at line 121?).

Reply:
When the error is nil, it only means the CRD exists and we can get it successfully, but we can't tell if the CRD manifest is changed or not, so still need to run update action in following codes. If the manifest has no change, it will do nothing and ResourceVersion will keep the same. If the manifest is changed, it will update the CRD and ResourceVersion will be updated.

Reply:
I saw where the confusion comes from - I meant the `dri.Create` error, not the `dri.Get` error, and am asking should we add `if err == nil\n return` before this line.